py3-libclang.yaml

```diff
@@ -0,0 +1,40 @@
+# Generated from https://pypi.org/project/libclang/
+package:
+  name: py3-libclang
+  version: 16.0.6
+  epoch: 0
+  description: 'Clang Python Bindings, mirrored from the official LLVM repo: https://github.com/llvm/llvm-project/tree/main/clang/bindings/python, to make the installation process easier.'
```

Comment:
Not blocking, but I'd love to understand more about why this is easier, and maybe try to use LLVM's python bindings directly.